### PR TITLE
Adding Audio output icon in the sample app player screen.

### PR DESCRIPTION
**Pull request by hardikagr**
#### WHAT
Adding a button in the sample apps player screen that shows audio output as well as the volume.

#### WHY
The current volume button does not indicate the audio output the app is playing on hence it is not intutive that clicking on that button will open an UI to connect any suitable output.

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

**Comment by kul3r4** [comment]
Hello,
can you try to run commands from https://google.github.io/horologist/contributing/#development to try to fix the build?

**Comment by kul3r4** [comment]
Hello,
try to run commands from https://google.github.io/horologist/contributing/#development to fix the build

File: media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SetAudioOutputButton.kt [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import androidx.wear.compose.material.MaterialTheme
+import com.google.android.horologist.audio.ui.R
+import com.google.android.horologist.audio.ui.components.AudioOutputUi
+import com.google.android.horologist.audio.ui.VolumeUiState
+import com.google.android.horologist.compose.material.Icon
+import com.google.android.horologist.compose.material.IconRtlMode
+import com.google.android.horologist.images.base.paintable.DrawableResPaintable
+
+/**
+ * Button to launch a screen to control the system volume, using volume up icon as
```
Comment (kul3r4) [inline media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SetAudioOutputButton.kt:43]: can we add in this doc how this is different from setVolumeButton? Currently this doc is the same for both. 
e.g. say that this button also adds a badge to identify the current output device

File: media/sample/src/main/java/com/google/android/horologist/mediasample/ui/player/UampSettingsButtons.kt [hunk 2]
```diff
@@ -49,9 +51,10 @@ public fun UampSettingsButtons(
             enabled = enabled,
         )
 
-        SetVolumeButton(
+        SetAudioOutputButton(
```
Comment (kul3r4) [inline media/sample/src/main/java/com/google/android/horologist/mediasample/ui/player/UampSettingsButtons.kt:54]: Given we are stopping using "SetVolumeButton" can we add in the SetVolumeButton doc a link to this button saying "If you want to display the current selected output, use SetAudioOutputButton"

**Review by kul3r4 — APPROVED** [review]
Some doc suggestions

File: media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SetAudioOutputButton.kt [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+  iconAlignment: Alignment = Alignment.Center,
+  iconPadding: PaddingValues? = null,
+  badgeColor: Color = MaterialTheme.colors.primary,
+) {
+  SettingsButton(
+    modifier = modifier,
+    onClick = onVolumeClick,
+    enabled = enabled,
+    imageVector = when {
+      audioOutputUi?.isConnected ?: false -> audioOutputUi!!.imageVector
```
Comment (yschimke) [inline media/audio-ui/src/main/java/com/google/android/horologist/audio/ui/components/actions/SetAudioOutputButton.kt:64]: I suspect with `== true` that you won't need !!

File: datalayer/phone/api/current.api [hunk 4]
```diff
@@ -3,11 +3,11 @@ package com.google.android.horologist.datalayer.phone {
 
   @com.google.android.horologist.annotations.ExperimentalHorologistApi public final class PhoneDataLayerAppHelper extends com.google.android.horologist.data.apphelper.DataLayerAppHelper {
     ctor public PhoneDataLayerAppHelper(android.content.Context context, com.google.android.horologist.data.WearDataLayerRegistry registry);
-    method public com.google.android.horologist.data.AppHelperResultCode? checkCompanionVersionSupportTileEditing();
+    method public error.NonExistentClass! checkCompanionVersionSupportTileEditing();
```
Comment (yschimke) [inline datalayer/phone/api/current.api:6]: This looks wrong. How did you generate this?

Comment (hardikagr) [reply datalayer/phone/api/current.api]: I was running `./gradlew spotlessApply spotlessCheck compileDebugSources compileReleaseSources metalavaGenerateSignature metalavaGenerateSignatureRelease lintDebug`, Although after updating my jvm version the same command is producing a more intutive result now.

File: media/sample/src/main/java/com/google/android/horologist/mediasample/ui/player/UampSettingsButtons.kt (hunk 2 above)
Comment (hardikagr) [reply media/sample/src/main/java/com/google/android/horologist/mediasample/ui/player/UampSettingsButtons.kt:54]: Kindly check the code's comment once.

**Comment by yschimke** [comment]
@hardikagr can you add a screenshot test for either the component states, or one for a uamp screen containing it.

We should have tests for each new composable.

**Comment by yschimke** [comment]
@hardikagr This can be in an immediate follow up if you would prefer to land this first?

**Comment by yschimke** [comment]
@hardikagr Are you still planning to add a test?

**Pull request merged**